src/lib/rnp.cpp

```diff
@@ -2099,9 +2099,6 @@ try {
     if (!input || !buf) {
```

Comment:
Please also allow to work with NULL input buffer and zero input size, and add the corresponding test.